### ISSUE TITLE
Doesn't Support PRs from Forked Repos

### ISSUE THREAD

**Issue by randyesq** [nemccarthy/stash-pullrequest-builder-plugin#90]
This is more of a question than an issue. Does this plugin support running a build for a pull request between a fork of a repo back to the original repo? If it does, I can't seem to figure it out and maybe there is a request for more documentation around the subject.

**Comment by nickbroon** [comment]
Yes it does. I'd imagine that's it's primary use case for most people.
Can you provide more details of what is not working for you.

**Comment by randyesq** [comment]
I just can't get it to build when I attempt a PR from a fork back to the original repository. The Jenkins log doesn't seem to pick up anything changing for my PR job.

**Comment by nickbroon** [comment]
I'd suggest creating a Jenkins Logger for `stashpullrequestbuilder` and level `FINEST` and seeing there is anything useful in there.
